helpers/embeds.js

```diff
@@ -0,0 +1,65 @@
+import { html, nothing, render } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { map } from 'lit/directives/map.js';
+import { tryGetPluginByKey } from './plugins.js';
+
+const embedTypeAttributeName = 'data-d2l-embed-type';
+
+export async function createEmbedPlaceholder(embedType, props) {
```

Comment:
Not to bike shed, but I am wondering is we want to replace "Placeholder" term with "Template" here. Whenever I think of the placeholder, I think if the thing that the html-editor displays in its design-view, which I expect is different, ya? Might help avoid confusion when we're talking about these things.

Reply:
I don't personally have strong opinions here, so I'm fine with whatever we think is ideal. I don't really think it's bikeshedding either, since this is going to be a public (at least to D2L) API. 😆

Reply:
Does that make pie filling a placeholder? It fills a place and it's temporary because it's destined to be eaten. 🤔

Reply:
At least for myself, it would be helpful to somehow differentiate. If it's a matter of calling it DesignViewPlaceholder in the editor context, and Placeholder here, that would be sufficient as along as we're are consistent.

Reply:
Before you commented, I did update it to `createEmbedViewPlaceholder` (with the implied  assumption that the editor one would be `createEmbedEditPlaceholder`), though I'm not fixed to that either. I do think that a distinction is necessary if we're switching to having the edit one be centralized as well (which I think is where we want to go). I'm content with whatever naming we all feel like, though I wonder how prevalent the "Design view" parlance is w.r.t. the editor.

Reply:
I know "design mode" or "design view" is something we've used before to draw that distinction, but we've never really had need to expose it externally. The C+ stuff still remains the only set of plugins that weren't written by either you or me, so I suspect there's no prevalent terminology for this stuff. 😆